test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4683,18 +4683,15 @@ class Server(PBSService):
 
     actions = ExpectActions()
 
-    # default attribute dictionary when user unset any attribute
-    revert_default_attrib = {'scheduling': True, 'resv_enable': True,
-                             'max_array_size': 10000,
-                             'eligible_time_enable': False,
-                             'max_concurrent_provision': 5,
-                             'log_events': 511, 'mail_from': 'adm',
-                             'query_other_jobs': True,
-                             'scheduler_iteration': 600,
-                             'resources_default.ncpus': 1,
-                             'pbs_license_min': 0,
-                             'pbs_license_max': 2147483647,
-                             'pbs_license_linger_time': 31536000}
+    server_ignore_attrs = [ATTR_SvrHost, ATTR_total, ATTR_count, ATTR_managers,
+                           ATTR_operators, ATTR_dfltque, ATTR_FlatUID,
+                           ATTR_FLicenses, ATTR_nodefailrq,
+                           ATTR_pbs_license_info, ATTR_license_count,
+                           ATTR_version, ATTR_power_provisioning,
+                           ATTR_DefaultChunk + '.ncpus',
+                           ATTR_max_job_sequence_id, ATTR_status, 'id']
```

Comment:
Why are you ignoring these attributes ? Not all in this list are unsettable. For example, we can unset flatuid, node_fail_requeue, pbs_license_info, default_chunk.ncpus, max_job_sequence_id. All these values will be removed when unset. Remove these from the list and add an else condition for if condition at line number 8474